content/en/docs/tasks/administer-cluster/reserve-compute-resources.md

```diff
@@ -114,7 +115,8 @@ daemons.
 
 To optionally enforce `system-reserved` on system daemons, specify the parent
 control group for OS system daemons as the value for `--system-reserved-cgroup`
-kubelet flag.
+kubelet flag. If `system-reserved` is enforced, kubelet will evict processes 
+of the specified cgroup if the threshold is exceeded.
```

Comment:
What does “evict” mean here?

Reply:
The kubelet won't "evict" processes - the system will starve them of CPU/OOMKill them to keep them within limits.

It does this by setting limits on the existing cgroups to whatever kube-reserved/system-reserved are set to (https://github.com/kubernetes/kubernetes/blob/3ffdfbe286ebcea5d75617da6accaf67f815e0cf/pkg/kubelet/cm/node_container_manager_linux.go#L114-L133).

Reply:
If the kubelet won't evict processes, I suggest this sentence be re-phrased